docs/settings/security-settings.asciidoc

```diff
@@ -50,118 +46,75 @@ The valid settings in the `xpack.security.authc.providers` namespace vary depend
-`<provider-name>.showInSelector` {ess-icon}
-| Flag that indicates if the provider should have an entry on the Login Selector UI. Setting this to `false` doesn't remove the provider from the authentication chain.
-
-2+a|
-[TIP]
-[NOTE]
-============
-You are unable to set this setting to `false` for `basic` and `token` authentication providers.
-============
-
-| `xpack.security.authc.providers.<provider-type>.`
-`<provider-name>.accessAgreement.message` {ess-icon}
-| Access agreement text in Markdown format. For more information, refer to <<xpack-security-access-agreement>>.
-
-| [[xpack-security-provider-session-idleTimeout]] `xpack.security.authc.providers.<provider-type>.`
-`<provider-name>.session.idleTimeout` {ess-icon}
-| Ensures that user sessions will expire after a period of inactivity. Setting this to `0` will prevent sessions from expiring because of inactivity. By default, this setting is equal to <<xpack-session-idleTimeout, `xpack.security.session.idleTimeout`>>.
-
-2+a|
-[TIP]
-============
-Use a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h', '7d', '1w').
-============
-
-| [[xpack-security-provider-session-lifespan]] `xpack.security.authc.providers.<provider-type>.`
-`<provider-name>.session.lifespan` {ess-icon}
-| Ensures that user sessions will expire after the defined time period. This behavior is also known as an "absolute timeout". If
-this is set to `0`, user sessions could stay active indefinitely. By default, this setting is equal to <<xpack-session-lifespan, `xpack.security.session.lifespan`>>.
+==== Valid settings for all authentication providers
+
+xpack.security.authc.providers.<provider-type>.<provider-name>.enabled {ess-icon}::
+Determines if the authentication provider should be enabled. By default, {kib} enables the provider as soon as you configure any of its properties.
+
+xpack.security.authc.providers.<provider-type>.<provider-name>.order {ess-icon}::
+Order of the provider in the authentication chain and on the Login Selector UI.
+
+xpack.security.authc.providers.<provider-type>.<provider-name>.description {ess-icon}::
+Custom description of the provider entry displayed on the Login Selector UI.
+
+xpack.security.authc.providers.<provider-type>.<provider-name>.hint {ess-icon}::
+Custom hint for the provider entry displayed on the Login Selector UI.
+
+xpack.security.authc.providers.<provider-type>.<provider-name>.icon {ess-icon}::
+Custom icon for the provider entry displayed on the Login Selector UI.
+
+xpack.security.authc.providers.<provider-type>.<provider-name>.showInSelector {ess-icon}::
+Flag that indicates if the provider should have an entry on the Login Selector UI. Setting this to `false` doesn't remove the provider from the authentication chain.
 
-2+a|
-[TIP]
-============
-Use a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h', '7d', '1w').
-============
+NOTE: You are unable to set this setting to `false` for `basic` and `token` authentication providers.
```

Comment:
This note should be indented under `xpack.security.authc.providers.<provider-type>.<provider-name>.showInSelector`